### DISCUSSION TITLE
Media Upload Feature

### DISCUSSION THREAD

**Discussion by momin-riyadh** [payloadcms/payload#826] [Q&A] [answered]
For publishing blog posts, we should need to add the MEDIA (image, video, sound, etc) feature in a text editor

**Comment by denolfe** [comment, marked as answer]
Hey @momin-riyadh , if I'm understanding you correctly - this is already possible. In your blog post collection, you can make the relationship field to your media required by adding `required: true` like this:

```ts
{
  name: 'image',
  type: 'upload',
  required: true, // Add this
  relationTo: 'media',
},
```

**Reply by momin-riyadh** [reply]
Thanks, denolfe! I'm a beginner and exploring
